RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,119 @@
+**Closed - Completed** - The bounty has been fully completed and paid out by the GMC. The bounty is no longer eligible for completion.  
+**Closed - Expired** - The bounty has met its expiration date. The bounty is no longer eligible for completion.  
+**Closed - Withdrawn** - The bounty has been withdrawn by the GMC. The bounty is no longer eligible for completion.  
+
+### Bounty Incentive Levers
+Bounty Incentive Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivise the generation and successful completion of bounty proposals.
+
+#### Lever Incentives
+* The incentive attached to each lever MAY be set by the GMC. 
+* Levers MAY have an absolute (fixed amount) component and/or a relative (percentage of total bounty payout) component. 
+* The payout of incentives for a given bounty MUST NOT reduce the bounty completion payout. 
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favours incentive recipients.
+* The total cost of lever incentives is not fixed. The GMC SHOULD set lever incentives with caution when funds are limited. 
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars, and MUST be paid out in one of the GMC's accepted payout token options selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are recieved by the GMC.
+* This incentive MUST be paid to the proposal author for a bounty proposal that is adopted by the GMC. 
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to recieve this incentive.
+
+#### Lever B: Definition Incentive
+The definition incentive lever aims to influence the quality and completeness of bounty definitions accompanying bounty proposals.
+* This incentive MUST be paid to the definiton author for bounty definitions provided alongside a bounty proposal that is adopted by the GMC. 
+* This incentive payout SHOULD be paid a maximum once per proposal, even if the bounty proposal is split into multiple definitions.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on definition quality and completeness.
+* This incentive SHOULD go to the proposal author unless an alternative is explicitly preapproved by the GMC adminitrator.
+
+#### Lever C: Completion Incentive
+The completion incentive lever aims to influence the quality and consistency of bounty work submitted by bounty hunters. It aims to do this by rewarding contacts that support, oversee and assist bounty hunters meet the requirements of open bounties. 
```

Comment:
Is there a way to require contacts be submitted with a bounty submission? Otherwise doesn't it add labor for the bounty hunter to identify all the people that helped them? Or were you imagining this would be handled entirely by the GMC during their regular meetings? (like who all helped this bounty hunter, lets add them to the list)

Reply:
They are required to be submitted as part of the bounty definition.